### ISSUE TITLE
Option to ignore certificate legitimacy

### ISSUE THREAD

**Issue by lennartbrandin** [jeffvli/feishin#14]
### What am I trying to achieve? 
I want to access a local navidrome server which is secured by a selfsigned certificate.

![image](https://user-images.githubusercontent.com/92579517/215225454-bcc02652-82cc-4732-9e1b-a49fc7be50b1.png)

 The same happens when using mismatching certificates (Domain differs from certificate domain).

### How could it be solved?
Adding an option to accepts self signed certificates (This is how i use it in a mobile app [Ultrasonic](https://gitlab.com/ultrasonic/ultrasonic))

### Whats the impact of not solving this?
The app remains unusable without publicly exposing the server or doing some nasty workarounds with host files.

**Comment by jeffvli** [comment]
I've added a toggle to ignore CORS for all servers.

![image](https://user-images.githubusercontent.com/42182408/216852698-1e2ac6df-54d5-48ba-a837-6e2d4b0f6749.png)

Can you test it on #23 to see if it's working for you? (Desktop executables will be posted on the pull request once the build is finished).

**Comment by lennartbrandin** [comment]
So i checked the ignore, restart but
self signed certificates do **not** work, i can see:
```
./Feishin-0.0.1-alpha4-linux-x86_64.AppImage 
libva error: /usr/lib/dri/iHD_drv_video.so init failed
11:20:07.051 › Checking for update
11:20:07.852 › Update for version 0.0.1-alpha4 is not available (latest version: 0.0.1-alpha4, downgrade is disallowed).
11:20:07.854 › checkForUpdatesAndNotify called, downloadPromise is null
[3484:0206/112030.097643:ERROR:cert_verify_proc_builtin.cc(670)] CertVerifyProcBuiltin for navidrome.domain failed:
----- Certificate i=0 (O=Internet Widgits Pty Ltd,ST=Some-State,C=DE) -----
ERROR: No matching issuer found
```
i also tested a domain mismatching certificate (which is issued by an authority) but that does not work either, and does not result in any console output.

**Comment by lennartbrandin** [comment]
Fixed in https://github.com/jeffvli/feishin/pull/23
